### DISCUSSION TITLE
IDE alternative

### DISCUSSION THREAD

**Discussion by Dayeneriss** [Cyfrin/foundry-full-course-cu#1779] [Ideas]
Hello, I'm Dayeneriss. Very happy to learn with Cyfrin. 
This week, i discovered 2 IDE seem cool : cursor @https://cursor.sh/ , and chainIDE @https://chainide.com/ , spe for builders in web3. Maybe it would be great to test them instead VScode. Tell me what you think about them.

**Comment by EngrPips** [comment]
Hey @Dayeneriss, Thanks for sharing what you found with us. I will try those and let you know how well I like them.

**Comment by dotunv** [comment]
The website for cursor is now https://cursor.com/. Thanks for the recommending chain ide

**Reply by EngrPips** [reply]
Excellent. Thanks for the heads-up, man.
